specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -3242,6 +3245,10 @@
         "snapshot": {
           "$ref": "#/definitions/ManagementPolicySnapShot",
           "description": "The management policy action for snapshot"
+        },
+        "version": {
```

Comment:
May I know why ManagementPolicyAction is required in a PolicyDefinition, but no one property in ManagementPolicyAction is required? 
So I can create a policy with empty action info, right?

Reply:
Yes, the properties within the actions are optional.